streaming/src/config/streaming_config.cc

```diff
@@ -37,6 +37,21 @@ void StreamingConfig::FromProto(const uint8_t *data, uint32_t size) {
   if (config.empty_message_interval() != 0) {
     SetEmptyMessageTimeInterval(config.empty_message_interval());
   }
+  if (config.flow_control_type() != proto::FlowControlType::UNKNOWN_FLOW_CONTROL_TYPE) {
+    SetFlowControlType(config.flow_control_type());
+  }
+  if (config.writer_consumed_step()) {
```

Comment:
`if (config.writer_consumed_step() != 0)` , keep the same with `config.empty_message_interval `